packages/node_modules/brightspace-auth-keys-dynamodb-store/src/public-key-store.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const AbstractPublicKeyStore = require('brightspace-auth-keys').AbstractPublicKeyStore;
+
+const clock = require('./clock');
+
+class DynamoPublicKeyStore extends AbstractPublicKeyStore {
+	constructor(client, tableName) {
+		super();
+		this._client = client;
+		this._tableName = tableName;
+	}
+
+	_storePublicKey(publicKey, expiresAt) {
+		return this._client.putItem(
+			{
+				TableName: this._tableName,
+				Item: {
+					'PublicKey': {
+						'S': publicKey
+					},
+					'ExpiresAt': {
+						'N': '' + expiresAt
+					}
+				}
+			}
+		).promise();
+	}
+
+	_lookupPublicKeys() {
+		const parameters = {
+			TableName: this._tableName,
+			FilterExpression: '#e > :expires_at',

```

Comment:
This is a predicate on inclusion, not eviction, correct?

Reply:
Correct. Keys are evicted by the table TTL

Reply:
I just meant more like, doublechecking it _should_ be a `>` and not a `<`

Reply:
yes, but the name is bad, I'm going to change that